lastFm-app/js/albumInfo.js

```diff
@@ -0,0 +1,122 @@
+/* jshint esversion: 6 */
+
+function AlbumInfo() {
+  this.method = "album.getinfo";
+  Request.apply(this, arguments);
+}
+
+AlbumInfo.prototype = Object.create(Request.prototype);
+AlbumInfo.prototype.constructor = AlbumInfo;
+
+AlbumInfo.prototype.getInfo = function(artistName, albumName) {
+  var queryObject = {
+    method: this.method,
+    artist: artistName,
+    album: albumName
+  };
+  
+  this.makeGet(queryObject, function(response) {
```

Comment:
your handler should be moved from `AlbumInfo`
`AlbumInfo` has responsibility to load data, but something else should draw it
the same for another instances